plugin/src/main/java/library.kt

```diff
@@ -1,4 +1,29 @@
+import com.stepango.forma.feature.applyFeatures
+import com.stepango.forma.feature.kotlinFeatureDefinition
+import com.stepango.forma.module.LibraryModule
+import com.stepango.forma.module.TestUtilModule
+import com.stepango.forma.module.UtilModule
+import com.stepango.forma.utils.applyDependencies
+import com.stepango.forma.validation.validate
+import com.stepango.forma.validation.validator
 import org.gradle.api.Project
 
-// Same as impl
-fun Project.library(): Unit = TODO()
\ No newline at end of file
+fun Project.library(
```

Comment:
How it's different from util?

Reply:
Interesting question) In our  first vision `library` should use only for kotlin sources and tests, but now we have same behaviour for `util`.  And differences are only in availability of other project dependencies. `Util` doesn't have it.